examples/triangle.zig

```diff
@@ -40,6 +40,11 @@ const vertices = [_]Vertex{
     .{ .pos = .{ -0.5, 0.5 }, .color = .{ 0, 0, 1 } },
 };
 
+// 48KB, if this is enough, the application won't need to fallback to the dynamic allocator (no allocations)
+// This is a trade-off between fixed (comptime) stack memory and speed/reliability (no heap allocations)
+// My setup required only 44208 bytes, but I just rounded it up to the nearest 4KB step arbitrarily to keep a margin.
+const stack_buffer_size = 48000;
+
```

Comment:
That previous change is fine because it simplifies the implementation, but im not sure that this is warranted for. Its supposed to be an example so it doesn't really have to be highly optimized or anything.

Reply:
verbose_log is fine?